### PR TITLE
updating issue template to direct users to new live chat source

### PR DESCRIPTION
**Pull request by jdivock-stripe**
what's on the tin

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ here that you believe represent bugs with the loader, not the SDK itself.
 
 If you're having general trouble with the Terminal JS SDK or your Stripe integration,
 please reach out to us using the form at https://support.stripe.com/email or
-come chat with us at #stripe on freenode. We're very proud of our level of
+come chat with us on discord (https://discord.com/invite/stripe). We're very proud of our level of
```
Comment (nazli-stripe) [inline .github/ISSUE_TEMPLATE.md:7]: nit: `Discord`

**Review by jil-stripe — CHANGES_REQUESTED** [review]
There's a different URL/wording that we use, blocking this PR until I can find it

**Comment by jdivock-stripe** [comment]
@jil-stripe updated the copy, thanks!

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -4,8 +4,8 @@ here that you believe represent bugs with the loader, not the SDK itself.
 
 If you're having general trouble with the Terminal JS SDK or your Stripe integration,
 please reach out to us using the form at https://support.stripe.com/email or
-come chat with us at #stripe on freenode. We're very proud of our level of
-service, and we're more than happy to help you out with your integration.
+come chat  live with our developers in on https://stripe.com/go/developer-chat.
```
Comment (jil-stripe) [inline .github/ISSUE_TEMPLATE.md:7]:
```suggestion
come chat live with our developers on https://stripe.com/go/developer-chat.
```

**Review by jil-stripe — DISMISSED** [review]

**Review by jil-stripe — APPROVED** [review]

**Pull request merged**